src/label/labelStyle.ts

```diff
@@ -135,17 +136,25 @@ function getLabelText<TLabelDataIndex>(
     const labelFetcher = opt.labelFetcher;
     const labelDataIndex = opt.labelDataIndex;
     const labelDimIndex = opt.labelDimIndex;
+    const labelValue = opt.labelValue;
     const normalModel = stateModels.normal;
     let baseText;
+    let extendParams = null;
     if (labelFetcher) {
+        if (labelValue != undefined && labelValue != null) {
+            extendParams = {
+                interpolatedValue: labelValue
+            };
+        }
+
         baseText = labelFetcher.getFormattedLabel(
             labelDataIndex, 'normal',
             null,
             labelDimIndex,
             normalModel && normalModel.get('formatter'),
             interpolatedValue != null ? {
                 interpolatedValue: interpolatedValue
-            } : null
+            } : extendParams
```

Comment:
This can be simplified to be `{interpolatedValue: retrive2(interpolatedValue, labelValue}`. `retrive2` is from `import { retrieve2 } from 'zrender/src/core/util';`.

Reply:
Yes, `extendParams` is not longer needed.

Reply:
oh，got it

Reply:
Hi，Test Case Commited

Reply:
Hi, please review!